manual/opt.tex

```diff
@@ -153,9 +153,110 @@ \subsubsection{quartic}
+   %&   \texttt{stepsize} &  real     & 0--1 & 0.25  & Step size for moving parameters\\
+   &   \texttt{max\_relative\_change} &  real     & $>0$ & 10.0 & Allowed change in cost function\\
+   &   \texttt{max\_param\_change} &  real     & $>0$ & 0.3 & Allowed change in wave function parameter\\
+   &   \texttt{shift\_i} &  real     & $>0$ & 0.01 & Initial diagonal      stabilizer added to the Hamiltonian matrix\\
+   &   \texttt{shift\_s} &  real     & $>0$ & 1.00 & Initial overlap-based stabilizer added to the Hamiltonian matrix\\
+   &   \texttt{targetExcited} &  text   & yes, no & no & Target excited state\\
+   &   \texttt{omega} &  real   & $<0$ & & Energy shift used to target different excited state\\
+   &   \texttt{chase\_lowest} &  text   & yes, no & yes & Chase the lowest eigenvector in iterative solver\\
+   &   \texttt{chase\_closest} &  text   & yes, no & no & Chase the eigenvector closest to initial guess\\
+   &   \texttt{block\_lm} &  text   & yes, no & no & Use block linear method\\
+   &   \texttt{nblocks} &  integer   & $>0$ &  & \# of blocks in BLM\\
+   &   \texttt{nolds} &  integer   & $>0$ &  & \# of old update vectors used in BLM\\
+   &   \texttt{nkept} &  integer   & $>0$ &  & \# of eigenvectors to keep per block in BLM\\
+  \hline
+\end{tabularx}
+\end{center}
+\end{table}
+
+Additional information:
+\begin{itemize}
+  \item \texttt{shift\_i}.  This is the initial value of direct term added to the diagonal of the Hamiltonian matrix. 
+                            More stable but slower optimization with a large value. The used value is auto-adjusted by the optimizer. 
+  \item \texttt{shift\_s}.  This is the initial value of the stabilizer based on the overlap matrix added to the Hamiltonian matrix. 
+                            More stable but slower optimization with a large value. The used value is auto-adjusted by the optimizer. 
+  \item \texttt{omega}.     Since approximate wave function has a finite variance of energy, is has been proved that the $\omega$ value
+                            needs to be ``shifted down'' by the amount of the standard deviation of the energy to target the correct state. 
+  \item \texttt{nblocks}.   This is the number of blocks used in block LM. The amount of memory required to store LM matrices decreases
+                            with increased number of blocks. But the error introduced by BLM would increase with number of blocks.  
+  \item \texttt{nolds}.     In BLM, the inter-block correlation is accounted for by including a small number of wave function update vectors
+                            outside the block. Larger \texttt{nolds} would include more inter-block correlation and more accurate results, but 
+                            also higher memory requirements. 
+  \item \texttt{nkept}.     This is the number of update directions taken from each block in BLM. If all directions are taken in each block, 
```

Comment:
The parameter name uses 'kept', but the verb in the description is 'taken'.  Might more consistent to use 'kept' in the description.
What setting would be needed to keep all the directions?   Is it problem -dependent?

"fewer than 5 directions"